### PR TITLE
 Add textures for the pixel shaders

### PR DESCRIPTION
**Pull request by elviscapiaq**
- It was added the textures for Albedo, NormalMap and metalness-roughness.

### CODE REVIEW

**Review by wangra-google — APPROVED** [review]

**Review by apazylbe — COMMENTED** [review]
One question, otherwise lgtm

File: assets/basic/models/spheres/albedo.png
[the remark this answers is not in the document]
Comment (elviscapiaq) [reply assets/basic/models/spheres/albedo.png:1]: I would say that both are correct. On one hand, we might want to use this in the future. On the other hand, I simply chose this texture.

[the remark this answers is not in the document]
Comment (apazylbe) [reply assets/basic/models/spheres/albedo.png:1]: I think its slightly weird to use this texture on spheres, but no problem with it I guess

[the remark this answers is not in the document]
Comment (Keenuts) [reply assets/basic/models/spheres/albedo.png:1]: I suppose they are fine for the usecase, but why do a copy? you can directly use the altimeter ones without copy.

**Review by Keenuts — APPROVED** [review]
LGTM, just the file copy we could avoid.
I'll merge once the conflict is resolved.

File: assets/basic/models/spheres/albedo.png
[the remark this answers is not in the document]
Comment (elviscapiaq) [reply assets/basic/models/spheres/albedo.png:1]: Done.

**Comment by elviscapiaq** [comment]
I avoided copying files for textures and also rebased the PR.

**Pull request merged**